Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/colmartech_vendors.yml

```diff
@@ -1933,6 +1933,134 @@
       - id: CMMaskGas
       - id: CMMaskCoif
 
+# MILITARY POLICE
+- type: entity
+  parent: ColMarTechBase
+  id: ColMarTechMilitaryPoliceEquipment
+  name: ColMarTech Military Police Equipment Rack
+  description: An automated rack hooked up to a colossal storage of Military Police standard-issue equipment.
+  components:
+  - type: AccessReader
+    access:
+    - [ "CMAccessBrig" ]
+  - type: Sprite
+    sprite: _RMC14/Structures/Machines/VendingMachines/ColMarTech/clothing.rsi
+  - type: CMAutomatedVendor
+    job: CMMilitaryPolice
+    sections:
+    - name: Standard Equipment
+      takeAll: CMStandard
+      entries:
+      - id: RMCVendorBundleMilitaryPoliceApparel
+      - id: CMArmorM2MP
+      - id: CMMRE
+    - name: Handgun Case
+#     choices: { CMGunCase: 1 }
+      entries: []
+#     - id: M77 Gun case
+#     - id: Combat Revolver Gun case
+#     - id: M1984 Gun case
+    - name: Backpack
+      choices: { CMBackpack: 1 }
+      entries:
+      - id: CMBackpackSecurity
```

Comment:
```suggestion
```

Reply:
I`l leave it, because someone might want backpack but spawn with satchel.